extensions/amp-autocomplete/amp-autocomplete.md

```diff
@@ -0,0 +1,90 @@
+<!--
```

Comment:
Our extension generator doesn't reflect this, but extension docs have a different format now (#20684). Nothing major changes, just that attribute sets have to be tables (which should be entirely HTML) and each extension has frontmatter that specifies some metadata.

Reply:
Thanks for the heads up! I think it could be a good idea at some point to update the generator to reflect that

Reply:
Yeah, working on it! :)

Reply:
oh sweet, thanks for taking that on